frontend/public/index.html

```diff
@@ -4,18 +4,19 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="URDR timelogging" />
+    <!-- Use local bootstrap.min.css to avoid crossover-errors -->
     <link
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      href="/dist/bootstrap.min.css"
       rel="stylesheet"
-      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
-      crossorigin="anonymous"
     />
+<!-- At least on FF this code makes no difference.
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
       rel="stylesheet"
     />
+ -->
```

Comment:
If I delete my cache, it makes a difference. Then, I see another font, similar but different. This part of the code includes our font "Lato" which is a google font. It's part of SciLifeLab's design rules, that's why we use it. So we need this code.

Reply:
Ok! I will look into the warnings as well.